src/sentry/grouping/enhancement-configs/legacy@2019-03-12.txt

```diff
@@ -23,11 +23,41 @@ family:native package:**/Frameworks/KSCrash.framework/**             -app
 family:native package:**/Frameworks/SentrySwift.framework/**         -app
 family:native package:**/Frameworks/Sentry.framework/**              -app
 
-# Sentry internal functions in Cocoa SDK
+# Needed for versions < sentry-cocoa 6.3.0 and static linking.
```

Comment:
We left the legacy file intentionally untouched, especially since the old grouping strategy will not group properly for native frames anyway. Please only add the new rules to `common@2019-03-23.txt`.